### DISCUSSION TITLE
I currently use Vimeo Review

### DISCUSSION THREAD

**Discussion by bobbyrabbit** [elonen/clapshot#33] [Ideas]
Vimeo review is pretty simple and you are almost there with Clapshot, it just needs to be a little more user friendly with easy download options.
Just for your reference, Vimeo does everything I need it to do except I am limited by 20GB per week upload quota.

See this video [https://www.youtube.com/watch?v=AgzrilYzt_E](url) 

Send the viewer (collaborator) a review URL, from there they can add time coded messages and admin can reply if needed, very similar to Clapshot now (clapshot looks better). Admin can replace the video with the amendments, viewer can add more messages if needed to the replaced video. Or, they have the option to download the final uploaded video if happy.

**Comment by elonen** [comment]
By all means keep on using Vimeo or Frame.io or other cloud services. This is meant for the very specific case where you have to host the videos yourself for policy reasons.
